wafer/talks/views.py

```diff
@@ -52,7 +51,11 @@ def get_queryset(self):
         if self.request and Talk.can_view_all(self.request.user):
             talks = Talk.objects.all()
         else:
-            talks = Talk.objects.filter(Q(status=ACCEPTED) | Q(status=CANCELLED))
+            talks = Talk.objects.filter(
+                Q(status__in=(ACCEPTED, CANCELLED))
+                | Q(status__in=(SUBMITTED, UNDER_CONSIDERATION),
```

Comment:
Is there a reason this doesn't include Provisional as well?

Reply:
Good catch. Fixed.